examples/logger.py

```diff
@@ -3,20 +3,51 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
-
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
-
 from qiwis import BaseApp
 
+
+class Signaller(QObject):
+    """Signal for LoggingHandler."""
```

Comment:
We put a single blank line after the class docstring.
I wrote this in #2.

Therefore:
```python
class Signaller(QObject):
    """Signal ..."""

    signal = pyqtSignal(str)


class LoggingHandler(...):
...
```